### PR TITLE
Save answers even when there's no authored state

### PR DESCRIPTION
**Pull request by pjanik**
[#176576965]

Some interactives don't define any authored state.

Note that I changed how the "version" property is set in the report state. It's no longer taken from the authored state. I think it's a separate version / JSON format. Currently, it's a JSON with mode, authoredState, and interactiveState properties. It doesn't seem related to authored state format version that is interactive specific.

### CODE REVIEW

**Comment by pjanik** [comment]
I've just pushed a second, unrelated small change - `url_fragment` is now appended to iframe URLs. This property can be set by authors. Some interactives rely on query parameters for authoring and they didn't work in Activity Player.
LARA was appending url_fragment, but AP didn't.

File: src/components/activity-page/managed-interactive/managed-interactive.tsx [hunk 1]
```diff
@@ -48,6 +48,8 @@ export const ManagedInteractive: React.ForwardRefExoticComponent<IProps> = forwa
   const shouldWatchAnswer = isQuestion(props.embeddable);
   const [loading, setLoading] = useState(shouldWatchAnswer);
 
+  console.log(props.embeddable);
```
Comment (kswenson) [inline src/components/activity-page/managed-interactive/managed-interactive.tsx:51]: Remove?

**Review by kswenson — APPROVED** [review]
👍 Looks good

**Review by kswenson — APPROVED** [review]
👍 Looks good

File: src/components/activity-page/managed-interactive/managed-interactive.tsx (hunk 1 above)
Comment (pjanik) [reply src/components/activity-page/managed-interactive/managed-interactive.tsx]: Thanks, removed.

**Pull request merged**